content/en/agent/versions/upgrade_to_agent_v6.md

```diff
@@ -1,71 +1,69 @@
 ---
 title: Upgrade to Datadog Agent v6
-kind: guide
+kind: documentation
 aliases:
   - /agent/faq/upgrade-to-agent-v6
+  - /agent/guide/upgrade-to-agent-v6
 ---
 
-<div class="alert alert-info">
-To do a fresh install of Datadog Agent v6, refer to the Agent <a href="https://app.datadoghq.com/account/settings#agent">installation process</a>.
-</div>
+## Upgrade to Agent 6
```

Comment:
Should we add a note to the user for him/her to consider upgrading directly to Agent v7?

Reply:
👍